integration/README.md

```diff
@@ -20,4 +20,10 @@ Tests have one or more (possibly para-)virtualized machines made accessible with
 
 In this example, `single-node-slug-deploy` is a test case that will execute in the context of the virtual machine spawned by the given Vagrantfile. Consequently, executions are limited to OS X for the time being.
 
-Future versions of this may include the use of the [AWS provider in Vagrant](https://github.com/mitchellh/vagrant-aws) to host many nodes in a single p2 cluster.
\ No newline at end of file
+Future versions of this may include the use of the [AWS provider in Vagrant](https://github.com/mitchellh/vagrant-aws) to host many nodes in a single p2 cluster.
+
+## Running the integration tests
+
+1) Install Vagrant for osx
```

Comment:
While this works for our development environment, it's still an open source project, and we can't assume that everyone will be using VirtualBox on OS X.  Specifically, could you make these changes?
* Remove "for osx"
* Phrase (2) more as "if you're using VirtualBox, you will probably need this: ..."